test-common/eos-test-utils.c

```diff
@@ -701,6 +754,10 @@ update_commits (EosTestSubserver *subserver,
                            error))
         return FALSE;
 
+      /* No checksum means that we've already written out this commit's ref */
+      if (checksum == NULL)
+        continue;
```

Comment:
This block should move to the commit which adds the `get_current_commit_checksum()` call, above.